install_config/install/docker_registry.adoc

```diff
@@ -239,7 +239,17 @@ this repository via an image.
 To access the registry directly, such as to perform `docker push` or `docker pull` operations, you
 must first log in to the registry using an access token.
 
-. Ensure you are logged in to OpenShift as a link:../../architecture/core_concepts/projects_and_users.html#users[regular user]:
+. Create a link:../../architecture/core_concepts/projects_and_users.html#users[regular user], for example, with `HTPASSWD` authentication:
```

Comment:
Since they may have already created or have access to a regular user, or chosen a different identity provider, suggest something like:

----

If a link:../../architecture/core_concepts/projects_and_users.html#users[regular user] does not already exist, create one according to your chosen link:../../admin_guide/configuring_authentication.html[identity provider]. For example, if you are using `htpasswd` authentication: